ui/apps/platform/src/types/featureFlag.ts

```diff
@@ -15,4 +15,5 @@ export type FeatureFlagEnvVar =
     | 'ROX_SCANNER_V4'
     | 'ROX_VULN_MGMT_LEGACY_SNOOZE'
     | 'ROX_VULNERABILITY_ON_DEMAND_REPORTS'
+    | 'ROX_CUSTOMIZABLE_PLATFORM_COMPONENTS'
```

Comment:
@charmik-redhat can you move into order according to comment on line 2:
> add strings in alphabetical order to minimize merge conflicts when multiple people add strings.

Reply:
Done